test/unit/ngsiv2/examples/contextRequests/updateContextMultientityTimestampPlugin3.json

```diff
@@ -10,10 +10,10 @@
       }
```

Comment:
updateContextMultientityTimestampPlugin3.json was in the repository but it was never used. Thus, I have reused it (after the small adaptation shown in this PR) for the new test.

Reply:
NTC